client/my-sites/purchases/subscriptions/subscriptions-content.tsx

```diff
@@ -108,18 +106,24 @@ function NoPurchasesMessage() {
 	const translate = useTranslate();
 	const hasJetpackPartnerAccess = useSelector( hasJetpackPartnerAccessSelector );
 
-	let url;
 	if ( ! isJetpackCloud() ) {
-		url = selectedSite ? `/plans/${ selectedSite.slug }` : '/plans';
-	} else if ( hasJetpackPartnerAccess ) {
+		return (
+			<CompactCard className="subscriptions__list">
+				{ translate( 'You have made no purchases for this site.' ) }
```

Comment:
I wonder if the perfect tense for this verb (ie. "made") might risk being a little misleading. The list shows active purchases; it is possible that the user _has_ made purchases for that site, but that they're shown under the "Billing History" tab rather than the "Active Upgrades" tab.

```suggestion
				{ translate( 'You have no active upgrades for this site.' ) }
```

Feel free to ignore as perhaps I'm overthinking - but I could envisage this causing some confusion eg. if a recent purchase has just expired and a user is coming to renew it.

Reply:
I agree. I think this could be better worded.